pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/PinotTableIdealStateBuilder.java

```diff
@@ -206,6 +206,8 @@ public static void buildLowLevelRealtimeIdealStateFor(String realtimeTableName,
     if (idealState == null) {
       idealState = buildEmptyKafkaConsumerRealtimeIdealStateFor(realtimeTableName, nReplicas);
       create = true;
+    } else {
+      idealState.setReplicas(Integer.toString(nReplicas));
```

Comment:
I think the nReplicas will be set in PinotLLCRealtimeSegmentManager.java updateIdealState() line 371, can you please check that? If yes, revert the change for this file.